backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1192,3 +1192,21 @@ databaseChangeLog:
                   name: refreshed_at
                   type: DATETIME
                   remarks: The timestamp for the most recent refresh of this entity.
+  - changeSet:
+      id: add-is-deleted-to-api-user
+      author: jeremy.a.zitomer@omb.eop.gov
+      comment: Add soft deletion flag and auditing columns to API users table
```

Comment:
sigh, of all the changes i thought it'd be hard to work in post-merge, i didn't expect a comment on a liquibase changeset to be that 🗡️. now that i've merged to `main`, how do i merge in this suggestion without creating checksum conflicts? is it safe to add a `validCheckSums: ANY` to this changeSet?

Reply:
Given that it's merged to main and deployed to the DB, it's not clear that it's worth fixing, but the options _other_ than `ANY` (which is definitely not safe) are

1. add a comment with the "well actually" instead of changing the string—comments are not checksummed, so you can freely editorialize about past changes
2. update the changeset with the specific valid checksum that you want it to treat as still valid after you change the label—you can find that checksum in the databasechangelog table any place where this has been deployed, or you can just make the change and start up your dev copy of the app and wait until it crashes, at which point you can get the requisite information out of the stack trace. :grinning:

Reply:
lol jk just re-ran the app local with the comment changes and nothing broke *shrug*